securedrop_client/resources/__init__.py

```diff
@@ -63,3 +63,32 @@ def load_css(name):
     Return the contents of the referenced CSS file in the resources.
     """
     return resource_string(__name__, "css/" + name).decode('utf-8')
+
+
+def load_icon_button(normal: str, disabled=None, active=None, selected=None) -> QPushButton:
+    icon = QIcon()
+
+    pixmap = load_image(normal)
+    scaled = pixmap.scaled(18, 18, Qt.KeepAspectRatio, transformMode=Qt.SmoothTransformation)
```

Comment:
You'll notice I'm using `Qt.SmoothTransformation` for smoothing which uses bilinear filtering under the hood. Some of the svgs will need to be replaced because opening them directly in the browser shows that they are small and pixel-y so they will not scale nicely. But for now, the smoothing effect makes it look much better than using `FastTransformation` or not scaling.

Reply:
Looks like the dropdown arrow svg is just rendered horribly in my firefox applicaiton on linux as well as mac, so the problem wasn't how svgs were created and exported in Sketch, but rather how we're loading them into Qt widgets. After making a change to load svgs into icons using `addFile` or labels using `QSvgWidget` they now should look :sparkles: crispy (not blocky)